<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonStartGame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLoadGame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOption">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonQuit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Attaquer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Attaquer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Piocher">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Piocher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Afficher_carte">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>afficher votre main</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>   CardGame</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
